testing/e2e.sh

```diff
@@ -116,19 +133,23 @@ function test_bazel_build_then_run_docker_build_clean() {
   cd "${ROOT}"
   for target in $(bazel query 'kind("container_image", "testdata/...")');
   do
-    clear_docker
-    bazel build $target
-    # Replace : with /
-    ./bazel-bin/${target/://}
+    if [[ $(SHOULD_SKIP $target) -eq 1 ]]; then
```

Comment:
You should do a custom bazel query that removes the targets you dont need.
e.g., I ran this successfully :
bazel query 'kind("container_image", "testdata/..." except ("//testdata:absolute_data_path_image" union "//testdata:with_user" union "//testdata:dashdash_entrypoint"))' )

Reply:
Done.

Had to duplicate some code... It's also ugly to make the additional part of the query a constant string, and I can't do simply join on a list in bash like targetsToSkip.join(" union ") either...